corehq/apps/domain/views.py

```diff
@@ -564,6 +565,21 @@ def requeue_repeat_record(request, domain):
     return HttpResponse('OK')
 
 
+@require_POST
+@require_can_edit_web_users
+def send_mock_repeater_report(request, domain):
+    try:
+        email_id = request.POST.get('email_id')
+        repeater_id = request.POST.get('repeater_id')
+        data = csv.reader(request.FILES['payload_ids_file'])
+        payload_ids = [row[0] for row in data]
+        send_mock_repeater_payloads(repeater_id, payload_ids, email_id)
```

Comment:
Might wanna make this an async task, particularly since it doesn't return anything in this view.